code/modules/organs/organ_objects.dm

```diff
@@ -14,6 +14,7 @@
 	var/datum/organ/internal/organ_data       // Stores info when removed.
 	var/prosthetic_name = "prosthetic organ"  // Flavour string for robotic organ.
 	var/prosthetic_icon                       // Icon for robotic organ.
+	var/uber = 0
```

Comment:
Yeah, put that on account of my exhaustion. Now that I've finished the changelog I'll start crushing those issues apart.